webdriver/tests/bidi/browsing_context/reload/frame.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+pytestmark = pytest.mark.asyncio
+
+PAGE_CONTENT = "<div>foo</div>"
+
+
+async def navigate_and_assert(bidi_session, context, url):
```

Comment:
Is there a reason why this code has been moved out to its own method now? It's used only once. So I do not see a reason for.

Reply:
I was intending to add more methods and reuse those. Just inlined them for now.